packages/clerk-js/src/ui/elements/Action/ActionCard.tsx

```diff
@@ -26,7 +26,6 @@ export const ActionCard = (props: ActionCardProps) => {
           boxShadow: t.shadows.$actionCardShadow,
           gap: t.space.$4,
           borderRadius: t.radii.$lg,
-          overflow: 'hidden',
```

Comment:
Seems logical, @desiprisg do you remember why we added this in the first place ?

Reply:
Can't think of anything right now. Nothing seems broken. Makes sense to remove this.